build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -502,3 +502,10 @@ secondaryNetwork:
   {{- toYaml . | nindent 4 }}
   {{- end }}
 {{- end }}
+
+hostNetworkAcceleration:
+{{- with .Values.hostNetworkAcceleration }}
+  # Enable to accelerate Pod-to-Pod traffic in the Node's host network using nftables flowtable when traffic mode is
+  # noEncap or hybrid. External-to-Service and Pod-to-external traffic will be supported in the future.
```

Comment:
I don't recall if there was a review comment about this previously, but I am not sure that including this information is useful at this stage:

> External-to-Service and Pod-to-external traffic will be supported in the future.

I think mentioning this in the design doc, and in the commit message, is enough

Reply:
Removed